src/content/decentralized-identity.md

```diff
@@ -0,0 +1,157 @@
+
+Decentralized identifiers are issued, held, and controlled by individuals. An [Ethereum account](/developers/docs/accounts/) is an example of a decentralized identifier. Anyone is free to create as many accounts as they wish, without permission from anyone, and without need to store this identifier in a central registry.
+
+DIDs are stored on distributed ledgers (blockchains) or peer-to-peer networks. This makes DIDs [globally unique, resolvable with high availability, and cryptographically verifiable](https://w3c-ccg.github.io/did-primer/). A DID can be associated with different entities, including people, organizations, or government institutions.  
+
+## What makes this possible? {what-makes-decentralized-identifiers-possible} 
+
+### 1. Public Key Infrastructure (PKI) {#public-key-infrastructure} 
+
+Public-key infrastructure (PKI) is an information security measure that generates a [public key](/glossary/#public-key) and [private key](/glossary/#private-key) for an entity. Public-key cryptography is used in blockchain networks to authenticate user identities and prove ownership of digital assets. 
+
+A DID, like an Ethereum account, will have a private key and a public key. While the private key is used for encrypting and signing messages, the public key authenticates the identity of the entity controlling a DID. PKI provides proofs needed to authenticate entities and prevent impersonation and use of fake identities.
+
+### 2. Decentralized datastores {#decentralized-datastores}
+
+A blockchain serves as a verifiable data registry; an open, trustless, and decentralized repository of information. The existence of public blockchains, like Ethereum, eliminates the need to store identifiers in centralized registries. 
+
+If anyone needs to confirm the validity of a DID, they can look up the associated public key on the blockchain. This is different from traditional identifiers that can only be authenticated by third parties. 
+
+## How do decentralized identifiers and attestations enable decentralized identity? {#how-decentralized-identifiers-and-attestations-enable-decentralized-identity} 
+
+Decentralized identifiers (DIDs) and attestations are the major building blocks of decentralized identity. Decentralized identity or self-sovereign identity is the idea that identity-related information should be self-controlled, private, and portable. 
+
+In the context of decentralized identity, attestations (also known as [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/)) are tamper-proof, cryptographically verifiable claims made by the issuer. Every attestation or Verifiable Credential an entity (e.g., an organization) issues is associated with their DID. 
```

Comment:
> cryptographically verifiable

we shouldn't assume users know what this means

Reply:
The only alternative would be to take out "cryptographically", but that'd like bury the point (i.e., claims that are verifiable with cryptographic keys). Again, it's a question of whether simplifying some concepts further is necessary. 

I do think the "cryptographically verifiable" concept is explained in other parts, so someone who's read the entire thing should grasp it. For example: *"Because DIDs are stored on the blockchain, anyone can verify the validity of an attestation by cross-checking the issuer's DID on Ethereum.*"

Reply:
I would keep it in... at this point we've already introduced PKI. 
If needed, we could maybe just add a note to the section above where we talk about private/public keys and mention this. ie:
> PKI provides proofs needed to authenticate entities and prevent impersonation and use of fake identities, using [cryptographic signatures](https://andersbrownworth.com/blockchain/public-private-keys/) to verify all claims.

Reply:
I would keep it in... at this point we've already introduced PKI. 
If needed, we could maybe just add a note to the section above where we talk about private/public keys and mention this. ie:
> PKI provides proofs needed to authenticate entities and prevent impersonation and use of fake identities, using [cryptographic signatures](https://andersbrownworth.com/blockchain/public-private-keys/) to verify all claims.

Reply:
> I would keep it in... at this point we've already introduced PKI. If needed, we could maybe just add a note to the section above where we talk about private/public keys and mention this. ie:
> 
> > PKI provides proofs needed to authenticate entities and prevent impersonation and use of fake identities, using [cryptographic signatures](https://andersbrownworth.com/blockchain/public-private-keys/) to verify all claims.

Added this now.